src/Functions/concat.cpp

```diff
@@ -46,25 +46,30 @@ class ConcatImpl : public IFunction
 
     DataTypePtr getReturnTypeImpl(const DataTypes & arguments) const override
     {
-        if (arguments.size() < 2)
-            throw Exception(
-                ErrorCodes::TOO_FEW_ARGUMENTS_FOR_FUNCTION,
-                "Number of arguments for function {} doesn't match: passed {}, should be at least 2",
-                getName(),
-                arguments.size());
+        if (arguments.size() == 1)
```

Comment:
Can we return the same string if there's only one argument?

Reply:
No. We can't. Refer to https://github.com/ClickHouse/ClickHouse/pull/57000